pkg/odo/cli/component/push.go

```diff
@@ -175,6 +176,28 @@ func (po *PushOptions) Validate() (err error) {
 	// If Devfile is present we do not need to validate the below S2I checks
 	// TODO: Perhaps one day move Devfile validation to here instead?
 	if util.CheckPathExists(po.DevfilePath) {
+		spinner := log.Spinner("Validating K8s Resources Name")
```

Comment:
What about just `Validating devfile component` as the spinner text, like `odo create`?

Reply:
updated